### PR TITLE
U8 transforms 2

### PR DESCRIPTION
**Pull request by tdaede**

### CODE REVIEW

File: src/transform/inverse.rs [hunk 1]
```diff
@@ -810,10 +812,10 @@ trait InvTxfm2D: Dim {
     Self::inv_txfm2d_add_rs(input, output, stride, tx_type, bd);
   }
 
-  fn inv_txfm2d_add_rs(
-    input: &[i32], output: &mut [u16], stride: usize, tx_type: TxType,
+  fn inv_txfm2d_add_rs<T>(
```
Comment (ycho) [inline src/transform/inverse.rs]: Do you know where this function figure out the size of tx?  Would Self::W and Self::H be that info?

Comment (tdaede) [reply src/transform/inverse.rs]: Yes, exactly.

File: src/util.rs [hunk 2]
```diff
@@ -207,9 +207,12 @@ pub fn clamp<T: PartialOrd>(input: T, min: T, max: T) -> T {
   }
 }
 
-use num_traits::PrimInt;
+use num_traits::*;
 use std::mem::size_of;
 
+pub trait Pixel: PrimInt + Into<u32> + Into<i32> + AsPrimitive<i32> + 'static {}
+impl<T> Pixel for T where T: PrimInt + Into<u32> + Into<i32> + AsPrimitive<i32> + 'static {}
```
Comment (lu-zero) [inline src/util.rs]: Wouldn't be better 
``` rust
impl Pixel for u8 {}
impl Pixel for u16 {}
```

Comment (tdaede) [reply src/util.rs]: Ah yeah it would.

**Comment by tdaede** [comment]
OK. I'm going to rebase this on 64x64 tx and move them all over at once.

**Comment by tdaede** [comment]
Can you look over it a gain? Should be no longer wip

**Pull request merged**